erpnext/accounts/doctype/purchase_invoice/test_purchase_invoice.py

```diff
@@ -230,6 +230,23 @@ def check_gle_for_pi(self, pi):
 			self.assertEqual(expected_values[gle.account][1], gle.debit)
 			self.assertEqual(expected_values[gle.account][2], gle.credit)
 
+	def test_purchase_invoice_with_exchange_rate_difference(self):
```

Comment:
Let's also test the GL Entry Amount too in the test

Reply:
Done.